### ISSUE TITLE
torch.cat与torch.chunk的使用 | Hexo

### ISSUE THREAD

**Issue by Anthony-fei** [Anthony-fei/Anthony-fei.github.io#2]
https://anthony-fei.github.io/2020/07/30/torchcat-yu-torchchunk-de-shi-yong/ 

 torch.cat ( (A, B), dim=0)接受一个由两个（或多个）tensor组成的元组，按行拼接，所以两个（多个）tensor的列数要相同。 torch.cat ( (A, B), dim=1)是按列拼接，所以两个tensor的行数要相同。 torch.chunk(tensor, chunk_num, dim)与torch.cat()原理相反，它是将tensor按dim（行    或
